.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,165 @@
+---
+name: 🐛 Bug report
+about: Help us improving by reporting a bug
+labels: bug, 0. Needs triage
+---
+
+<!--
+Thanks for reporting issues back to Nextcloud!
+
+Note: This is the **issue tracker of Nextcloud**, please do NOT use this to get answers to your questions or get help for fixing your installation. This is a place to report bugs to developers, after your server has been debugged. You can find help debugging your system on our home user forums: https://help.nextcloud.com or, if you use Nextcloud in a large organization, ask our engineers on https://portal.nextcloud.com. See also  https://nextcloud.com/support for support options.
+
+Nextcloud is an open source project backed by Nextcloud GmbH. Most of our volunteers are home users and thus primarily care about issues that affect home users. Our paid engineers prioritize issues of our customers. If you are neither a home user nor a customer, consider paying somebody to fix your issue, do it yourself or become a customer.
+
+Guidelines for submitting issues:
+
+* Please search the existing issues first, it's likely that your issue was already reported or even fixed.
+    - Go to https://github.com/nextcloud and type any word in the top search/command bar. You probably see something like "We couldn’t find any repositories matching ..." then click "Issues" in the left navigation.
+    - You can also filter by appending e. g. "state:open" to the search string.
+    - More info on search syntax within github: https://help.github.com/articles/searching-issues
+    
+* This repository https://github.com/nextcloud/server/issues is *only* for issues within the Nextcloud Server code. This also includes the apps: files, encryption, external storage, sharing, deleted files, versions, LDAP, and WebDAV Auth
```

Comment:
Nop...

We need to create templates somehow that can easily be copied (manual or automatically) to other apps. Maybe similar to the labels

Reply:
it seems that you can also create a `.github` repository that contains default templates, in case the repo does not contain their own.

This would be especially useful, so we don't keep copy and pasting `CONTRIBUTING.md` and security policy.